src/_locales/en/messages.json

```diff
@@ -37,6 +37,49 @@
   "pay_error_notEnoughFunds": {
     "message": "Not enough funds to facilitate payment."
   },
+  "outOfFunds_error_title": {
+    "message": "Out of funds"
+  },
+  "outOfFunds_error_text": {
+    "message": "Funds have been depleted. You can no longer make payments."
+  },
+  "outOfFunds_error_textHint": {
+    "message": "Please add funds."
+  },
+  "outOfFunds_error_textDoNothing": {
+    "message": "If you do nothing, $AMOUNT$ will automatically be available on $AUTO_RENEW_DATE$",
+    "placeholders": {
+      "AMOUNT": { "content": "$1", "example": "$2.05" },
+      "AUTO_RENEW_DATE": { "content": "$2", "example": "Aug 22, 2024, 6:05 PM" }
+    }
+  },
+  "outOfFunds_action_optionOneTime": {
+    "message": "Let me add a one off payment"
+  },
+  "outOfFunds_action_optionRecurring": {
+    "message": "Let me add a monthly grant"
+  },
+  "outOfFundsAddFunds_label_walletAddress": {
+    "message": "Connected wallet address"
+  },
+  "outOfFundsAddFunds_label_amount": {
+    "message": "Amount"
+  },
+  "outOfFundsAddFunds_label_amountDescriptionOneTime": {
+    "message": "Enter the amount to add from the wallet."
+  },
+  "outOfFundsAddFunds_label_amountDescriptionRecurring": {
+    "message": "Enter the amount to add from the wallet. This amount will renew automatically every month (next: $NEXT_RENEW_DATE$}).",
+    "placeholders": {
+      "NEXT_RENEW_DATE": { "content": "$1", "example": "Aug 22, 2024" }
+    }
+  },
+  "outOfFundsAddFunds_action_addOneTime": {
+    "message": "Add funds"
+  },
+  "outOfFundsAddFunds_action_addRecurring": {
+    "message": "Add funds"
+  },
```

Comment:
Seeking a final pass for the copy shown @tselit @RabebOthmani

Reply:
"If you do nothing, $AMOUNT$ will automatically be available on $AUTO_RENEW_DATE$" - Just to make sure what we mean here is the amount setup previously would renew, let's say "If you do nothing, $AMOUNT$ will automatically be renewed and available on $AUTO_RENEW_DATE$"

I know that @tselit has a proposal on replacing one off payment and grant. This was also part of the conversation during the work week. I wonder if we should keep one off payment and grant until we update the overall copy. Equally we can go with Tseli's proposal and update later if needed.

Reply:
Everything else looks good to me.

Reply:
Thanks @sidvishnoi and @RabebOthmani!
I'm happy to with Rabi's suggestion to go with the current copy, which will be updated across the board when we revise the overall copy 👍🏾